packages/foreman/foreman-installer/foreman-installer.spec

```
@@ -146,6 +146,9 @@ done
 %{_sbindir}/foreman-proxy-certs-generate
 
 %changelog
+* Wed Aug 12 2020 Eric D. Helms <ericdhelms@gmail.com> - 1:2.2.0-0.6.rc1
+- Release foreman-installer 2.2.0
```

Not sure if it's important, but i think the message should be 
```suggestion
- Release foreman-installer 2.2.0-rc1
```
(also for the other packages)

---

Seems to be how we've done it in the past, based on the script. I'll have to add some functionality to do this in the future.